adobe/spectrum-css
# fix(button): use appropriate tokens for radii in buttons

## Description

Replaces all of occurrences of `--spectrum-component-pill-edge-to-text-*` (intended to be used as padding) in the button component with the appropriate `corner-radius-*` tokens.

## How and where has this been tested?

Verified locally in Storybook.

### Validation steps

1. Fetch the branch and run Storybook locally (or access the Storybook URL for the branch).
2. Navigate to the `button` component.
3. Verify that there are no `border-radius` regressions.

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:


5. If components have been modified, VRTs have been run on this branch:


## Screenshots

<img width="1419" alt="image" src="https://github.com/user-attachments/assets/17770ec1-d500-4a9a-bf1b-3f32da45f789">

## To-do list
